vocabs/v1/context.jsonld

```diff
@@ -1,90 +1,61 @@
-    "Bls12381G1Key2020": "sec:Bls12381G1Key2020",
-    "Bls12381G2Key2020": "sec:Bls12381G2Key2020",
-    "RsaVerificationKey2018": "sec:RsaVerificationKey2018",
-    "SchnorrSecp256k1VerificationKey2019": "sec:SchnorrSecp256k1VerificationKey2019",
-    "X25519KeyAgreementKey2019": "sec:X25519KeyAgreementKey2019",
-    "ServiceEndpointProxyService": "didns:ServiceEndpointProxyService",
-    "LinkedDomains": "https://identity.foundation/.well-known/resources/did-configuration/#LinkedDomains",
+
     "alsoKnownAs": {
       "@id": "https://www.w3.org/ns/activitystreams#alsoKnownAs",
       "@type": "@id",
       "@container": "@set"
     },
     "assertionMethod": {
-      "@id": "sec:assertionMethod",
+      "@id": "https://w3id.org/security#assertionMethod",
       "@type": "@id",
       "@container": "@set"
     },
     "authentication": {
-      "@id": "sec:authenticationMethod",
+      "@id": "https://w3id.org/security#authenticationMethod",
       "@type": "@id",
       "@container": "@set"
     },
     "capabilityDelegation": {
-      "@id": "sec:capabilityDelegationMethod",
+      "@id": "https://w3id.org/security#capabilityDelegationMethod",
       "@type": "@id",
       "@container": "@set"
     },
     "capabilityInvocation": {
-      "@id": "sec:capabilityInvocationMethod",
+      "@id": "https://w3id.org/security#capabilityInvocationMethod",
       "@type": "@id",
       "@container": "@set"
     },
     "controller": {
-      "@id": "sec:controller",
+      "@id": "https://w3id.org/security#controller",
       "@type": "@id"
     },
-    "created": {
-      "@id": "dc:created",
-      "@type": "xsd:dateTime"
-    },
-    "blockchainAccountId": "sec:blockchainAccountId",
     "keyAgreement": {
-      "@id": "sec:keyAgreementMethod",
-      "@type": "@id",
-      "@container": "@set"
-    },
-    "publicKey": {
-      "@id": "sec:publicKey",
+      "@id": "https://w3id.org/security#keyAgreementMethod",
       "@type": "@id",
       "@container": "@set"
     },
-    "publicKeyBase58": "sec:publicKeyBase58",
-    "publicKeyJwk": {
-      "@id": "sec:publicKeyJwk",
-      "@type": "@json"
-    },
     "service": {
-      "@id": "didns:service",
-      "@type": "@id",
-      "@container": "@set"
-    },
-    "serviceEndpoint": {
-      "@id": "didns:serviceEndpoint",
-      "@type": "@id"
-    },
-    "updated": {
-      "@id": "dc:modified",
-      "@type": "xsd:dateTime"
+      "@id": "https://www.w3.org/ns/did#service",
+      "@type": "@id",
+      "@container": "@set",
+      "@context": {
+        "@protected": true,
+        "id": "@id",
+        "type": "@type",
+        "serviceEndpoint": {
+          "@id": "https://www.w3.org/ns/did#serviceEndpoint",
+          "@type": "@id"
+        }
+      }
     },
     "verificationMethod": {
-      "@id": "sec:verificationMethod",
-      "@type": "@id"
+      "@id": "https://w3id.org/security#verificationMethod",
+      "@type": "@id",
+      "@container": "@set"
```

Comment:
This breaks compatibility with both [security-v2.jsonld](https://w3c-ccg.github.io/security-vocab/contexts/security-v2.jsonld) and the current [security-v3-unstable.jsonld](https://w3c-ccg.github.io/security-vocab/contexts/security-v3-unstable.jsonld), i.e. this leads to "protected term redefinition" JSON-LD errors.

Is the expectation that `verificationMethod` this will be changed in security-v3-unstable.jsonld as well? Couldn't this lead to problems with existing LD proof and VC implementations?

Reply:
Or is the assumption that the DID context and the security contexts will never be used together (I hope not).

Reply:
Good point @peacekeeper... we'll have to think hard about what we want here. 

I think what we're (slowly) learning is that having giant context files for security purposes is a double-edged sword. It's nice for testing and running experiments, but when you get to production, you really do want to protect the security terms and to be very specific about the security primitives you're pulling in. 

So, the current proposal is to do stuff like this for DID Documents (did:key example using 2020 suites -- note that we're pulling in a specific security suite, not the entire security context):

```
  "@context": [
    "https://www.w3.org/ns/did/v1",
    "https://w3id.org/security/suites/ed25519-2020/v1"
  ],
```

VCs would do more or less the same thing:

```
  "@context": [
    "https://www.w3.org/2018/credentials/v1",
    "https://w3id.org/security/suites/ed25519-2020/v1"
```

Example of the item above working:

https://tinyurl.com/yzp6alkg

Thoughts?